brainrender/atlas_specific/aba_gene_expression/api.py

```diff
@@ -93,9 +97,13 @@ def get_gene_experiments(self, gene_symbol):
             if isinstance(gene_symbol, int):  # it's an ID, get symbol
                 gene_symbol = self.get_gene_symbol_by_id(gene_symbol)
                 if gene_symbol is None:
-                    raise ValueError("Invalid gene_symbol argument")
```

Comment:
Is there a reason to handle this complicated nested statements here and not just failing in the method get_gene_symbol?

Reply:
nope, removed. thanks